client/my-sites/marketing/connections/service.jsx

```diff
@@ -414,6 +417,12 @@ export class SharingService extends Component {
 		return status;
 	}
 
+	getConnectionExpiry() {
```

Comment:
This isn't great, but the rest of this component is setup for having multiple connections to a service. We'd really need separate reconnection buttons in that case. 

This follows the pattern in `getConnectionStatus` that if one of the connections is broken then the status is `broken`. Here if the status is `refresh-failed` then we say that you need to complete the reconnection process by the earliest expiry date. It's all hypothetical at the moment as you can only have one Instagram connection.